arm-servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -2974,7 +2974,7 @@
     "EventHealthEvaluation": {
       "type": "object",
       "allOf": [{"$ref": "#/definitions/HealthEvaluation"}],
-      "x-ms-discriminator-value": "1",
+      "x-ms-discriminator-value": "EventHealthEvaluation",
```

Comment:
We decided to use a readable string, so user could read the type without the need to map it.